0800-appendix-197844-bighorn.Rmd

```diff
@@ -0,0 +1,97 @@
+---
+output: html_document
+editor_options: 
+  chunk_output_type: console
+  markdown: 
+    wrap: 72
+---
+
+```{r setup-197844, eval = F}
+knitr::opts_chunk$set(echo=FALSE, message=FALSE, warning=FALSE)
+source('scripts/packages.R')
+source('scripts/tables.R')
+
+```
+
+```{r  load-data-197844}
+my_site <- 197844
+
+```
+
+# Bighorn Creek - 197844 - Appendix {-}
+
+## Site Location {.unnumbered}
+
+PSCIS crossing `r as.character(my_site)` is located on a tributary to Bighorn Creek on the Cabin FSR. The site is located approximately 11km west of the junction with the Wigwam FSR at a point approximately 30k south of Morrissey. Cabin FSR(forest file ID 5466) is ultimately the responsibility of Ministry of Forests, Lands, Natural Resource Operations and Rural Development - Rocky Mountain Forest District. Although unconfirmed, Canfor may have a road use permit for this section of FSR.
```

Comment:
name of the ministries (MoE, FLNR) changed.  It is now Ministry of Forests. (MoE is some other like Land and Water stewardship I think)